runtime.js

```diff
@@ -161,11 +172,11 @@ export async function setContextMenu (o) {
     .entries(o)
     .flatMap(a => a.join(':'))
     .join('_')
-  return await ipc.send('context', o)
+  return ipc.send('context', o)
 }
```

Comment:
Why drop all the `await`?

Reply:
It's not necessary and can take an extra tick in old versions of WebKit

Reply:
Absolutely necessary when you want to preserve the call stack

see https://gist.github.com/jwerle/1bc93e303c3532abe0bdcbbea75e528d

Reply:
added again. BTW is this V8-only feature?